bot/across_the_stars_cmds.py

```diff
@@ -0,0 +1,33 @@
+from bot.across_the_stars.planets import Planets
+import discord
+from discord.ext import commands
+import os
+
+
+class AcrossTheStarsCmds(commands.Cog):
```

Comment:
Precisamos documentar nossas Cogs para que os comandos de ajuda consigam entender do se trata esse conjunto de comandos.
```suggestion
class AcrossTheStarsCmds(commands.Cog):
    """
    Jogo Across The Stars
    """
```